### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by josianerodrigues** [pangwong/pytorch-multi-label-classifier#3]
Hi @pangwong,
Thank you for sharing your project with us.
I saw that the loss function you're using for multilabel classification is CrossEntropyLoss function. I thought this function was only used for one-label classification. Could you explain how you used the CrossEntropyLoss function for multilabel classification? I'm trying to run my model for the NUSWIDE dataset using the MultiLabelSoftMarginLoss function, but the accuracy is getting pretty low.

**Comment by pangwong** [comment]
For a multi-label classification problem, we regard each label as a single classification problem, so the final loss is sum of CrossEntropyLoss of each label.  As for MultiLabelMarginLoss, it's similar with a combination of multiple HingeLoss for Multi-Class problem, also similar with SVM classifier.
